scipy/stats/_crosstab.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+
+def crosstab(*args, levels=None):
+    """
+    Create a table of the counts of the tuples in ``zip(*args)``.
+
+    When ``len(args) > 1``, the array computed by this function is
+    often referred to as a *contingency table* [1]_.
+
+    The arguments must be sequences with the same length.  The second return
+    value, `count` is an integer array with ``len(args)`` dimensions.  If
+    `levels` is None, the shape of `count` is ``(n0, n1, ...)``, where ``nk``
+    is the number of unique elements in ``args[k]``.
+
+    Parameters
+    ----------
+    args : sequences
+        A sequence of sequences whose unique aligned elements are to be
+        counted.  The sequences in args must all be the same length.
+    levels : sequence, optional
```

Comment:
I think it's worth pointing out that the default values for `levels` for input `i` are the output of `np.unique(args[i])`.   Specifically, that the order in which the values occur in `args` is irrelevant - the order in `levels` is sorted - for example, for strings, alphabetically.

Reply:
I added more about this to the docstring.